apps/site/pages/docs/multiple-page-templates.mdx

```diff
@@ -12,9 +12,29 @@ import { Callout } from 'nextra/components'
 
 Multiple page templates allow you to create flexible layouts in the Headless CMS for product and product listing pages (PDPs and PLPs) based on criteria like product information, user context, product type, brand, and template hierarchy. This means you can design and manage pages hierarchically instead of creating a single, static template for all products or listings.
 
-<Callout type="warning" emoji="⚠️">
-  This feature is already included in stores with the 3.x. `@faststore/core` package version. To update the `@faststore/core` package, refer to this [release note](https://github.com/vtex/faststore/releases/tag/v3.0.0).
-</Callout>
+
+## Before you begin
+
+To benefit from this feature, follow the steps below:
+
+1. Update your store’s `@faststore/core` package to the `v3.0.17` by running the following commands:
+
+```bash
+yarn add @faststore/core@3.0.17
+``` 
+```bash
+yarn add -D @faststore/cli@3.0.17
```

Comment:
This feature specifically doesn't change the cli, and the cli has no 3.0.17 version. 
https://www.npmjs.com/package/@faststore/cli?activeTab=versions

So, I recommend using the most recent in this case.

Reply:
Thanks! Fixed here: https://github.com/vtex/faststore/pull/2268/files#diff-05ed0a631665c06b7584525452a105b48461f81b9b3f816f7d2b346311b0163fR20-R28